cmd/gokeyless/gokeyless.go

```diff
@@ -54,31 +55,32 @@ func init() {
 func main() {
 	flag.Parse()
 
-	s, err := server.NewServerFromFile(certFile, keyFile, caFile, net.JoinHostPort("", port), "")
-	if err != nil {
-		log.Warningf("Could not create server. Running initializeServer to get %s and %s", keyFile, certFile)
-
-		// If an existing CSR (and associated key) exists, don't
-		// proceed/overwrite the key.
-		if _, err := os.Stat(csrFile); !os.IsNotExist(err) {
-			log.Fatalf("an existing CSR was found at %q: remove once a signed certificate has been installed.", csrFile)
-		}
-
+	init := needNewCertAndKey()
+	if init {
+		log.Info("Now let's get server certificate and key")
 		// Allow manual activation (requires the CSR to be manually signed).
 		if manualMode {
+			log.Info("[Manual mode]")
 			if err := manualActivation(); err != nil {
 				log.Fatal(err)
 			}
 
 			log.Infof("CSR generated (requires manual signing) and written to %q",
```

Comment:
Update this to encapsulate below: `"CSR generated (contact CloudFlare for manual signing) and written to %q`